contents/docs/libraries/android/index.mdx

```diff
@@ -50,6 +50,13 @@ PostHog.alias("distinct_id")
 
 We strongly recommend reading our docs on [alias](/docs/data/identify#alias-assigning-multiple-distinct-ids-to-the-same-user) to best understand how to correctly use this method.
 
+## Person Profiles ("anonymous" vs "identified" persons)
+The default behavior for this SDK is to use the `identified_only` mode of person processing, but this can be changed in the config. The options are:
```

Comment:
```suggestion
## Person Profiles ("anonymous" vs "identified" persons)

The default behavior for this SDK is to use the `IDENTIFIED_ONLY` mode of person processing, but this can be changed in the config. The options are:
```